RoddyCore/src/de/dkfz/roddy/execution/io/LocalExecutionService.groovy

```diff
@@ -72,6 +74,31 @@ public class LocalExecutionService extends ExecutionService {
         return outputStream;
     }
 
+    /**
```

Comment:
Also, there is a readfile method (or it has another name) in the executionservice classes. 
Why do you have the getJobStateInfoLine here but the getJobStateInfoLineForJob in the SSH service?
I'd rather like to see all the code in one class. Maybe JobStateLineHelper?